server/routers/advisor_router.py

```
@@ -0,0 +1,423 @@
+
+    # Load CV evaluation summary and optional job offer
+    latest_eval = _latest_evaluation_summary(db, current_user.id)
+    job_summary = None
+    if payload.context and payload.context.jobId:
+        try:
+            from server.methods.FT_job_search import get_offer_by_id
+            from server.routers.jobs_router import _flatten_offer
+            offer_data = await asyncio.to_thread(get_offer_by_id, payload.context.jobId)
+            flat = _flatten_offer(offer_data)
+            job_summary = _job_offer_summary(flat)
+        except Exception as e:
```

Ruff (0.14.14)

[warning] Do not catch blind exception: `Exception`

(BLE001)